Documentation/ApiOverview/CommandControllers/Index.rst

```diff
@@ -66,6 +66,9 @@ Command Class
 
 Example taken from :php:`ListSysLogCommand` in the core and simplified::
 
+    use Symfony\Component\Console\Command\Command;
+    use Symfony\Component\Console\Input\InputInterface;
+    use Symfony\Component\Console\Output\OutputInterface;
```

Comment:
If you want to be consistent you'll also need to import `InputArgument` in the example below.

Reply:
Good point, thanks. Done.